### DISCUSSION TITLE
Saving presets

### DISCUSSION THREAD

**Discussion by PampoenKoekie** [shteeve3d/blender-wiggle-2#54] [Ideas]
Saving presets would be a valuable addition to store a particular setting for a certain character or whatever it may be.

**Comment by schroef** [comment]
ill add it, i also did that with the other version. But since it has more settings now. Should it be divided into 2 sections or just everything. I think the last one. Ill also add a reset button, thats ease of use

**Comment by schroef** [comment]
I got it working already, it save both tail and head. Dont think we need those other settings from utilities
![save-presets](https://github.com/user-attachments/assets/8fe31240-7f87-4132-b158-158f40b612e3)
